base/applications/taskmgr/lang/pt-PT.rc

```diff
@@ -0,0 +1,443 @@
+    CONTROL "Bytes de gravação de E/S", IDC_IOWRITEBYTES, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 123, 127, 97, 10
+    CONTROL "Outras E/S", IDC_IOOTHER, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 123, 138, 47, 10
+    CONTROL "Outros bytes de E/S", IDC_IOOTHERBYTES, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 123, 149, 77, 10
+    CONTROL "Command &Line", IDC_COMMANDLINE, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 123, 160, 95, 10
+END
+
+/* String Tables */
+
+STRINGTABLE
+BEGIN
+    IDS_APP_TITLE "Gerenciador de tarefas"
+    IDC_TASKMGR "Gerenciador de tarefas"
+    IDS_IDLE_PROCESS "Processo inativo do sistema"
+END
+
+STRINGTABLE
+BEGIN
+    ID_FILE_NEW "Executa um novo programa"
+    ID_OPTIONS_ALWAYSONTOP "O 'Gerenciador de tarefas' permanece na frente de todas as outras janelas, a menos que seja minimizado"
+    ID_OPTIONS_MINIMIZEONUSE "O 'Gerenciador de tarefas' é minimizado toda vez que uma operação de alternância for executada"
+    ID_OPTIONS_HIDEWHENMINIMIZED "Oculta o 'Gerenciador de tarefas' quando ele estiver minimizado"
+    ID_VIEW_REFRESH "Força o 'Gerenciador de tarefas' a atualizar a tela agora, independentemente do parâmetro 'Velocidade de atualização'"
+    ID_VIEW_LARGE "Exibe as tarefas usando ícones grandes"
+    ID_VIEW_SMALL "Exibe as tarefas usando ícones pequenos"
+    ID_VIEW_DETAILS "Exibe as informações sobre cada tarefa"
+    ID_VIEW_UPDATESPEED_HIGH "Atualiza a tela duas vezes por segundo"
+    ID_VIEW_UPDATESPEED_NORMAL "Atualiza a tela a cada dois segundos"
+    ID_VIEW_UPDATESPEED_LOW "Atualiza a tela a cada quatro segundos"
+END
+
+STRINGTABLE
+BEGIN
+    ID_VIEW_UPDATESPEED_PAUSED "Não atualiza a tela automaticamente"
+    ID_WINDOWS_TILEHORIZONTALLY "Organiza as janelas lado a lado horizontalmente na área de trabalho"
+    ID_WINDOWS_TILEVERTICALLY "Organiza as janelas lado a lado verticalmente na área de trabalho"
+    ID_WINDOWS_MINIMIZE "Minimiza as janelas"
+    ID_WINDOWS_MAXIMIZE "Maximiza as janelas"
+    ID_WINDOWS_CASCADE "Organiza as janelas em cascata diagonalmente na área de trabalho"
+    ID_WINDOWS_BRINGTOFRONT "Coloca a janela em primeiro plano, mas não alterna o foco para ela"
+    ID_HELP_TOPICS "Exibe os tópicos da 'Ajuda' do 'Gerenciador de tarefas'"
+    ID_HELP_ABOUT "Exibe informações sobre o programa, o número da versão e os direitos autorais"
+    ID_FILE_EXIT "Sai do aplicativo 'Gerenciador de tarefas'"
+    ID_OPTIONS_SHOW16BITTASKS "Exibe as tarefas de 16 bits do Windows sob o arquivo ntvdm.exe associado"
+    ID_VIEW_SELECTCOLUMNS "Seleciona quais colunas serão visíveis na tela Processo"
+    ID_VIEW_SHOWKERNELTIMES "Exibe o tempo do núcleo nos gráficos de desempenho"
+    ID_VIEW_CPUHISTORY_ONEGRAPHALL "Apenas um único gráfico exibe o uso total de CPU"
+    ID_VIEW_CPUHISTORY_ONEGRAPHPERCPU "Cada CPU possui o seu próprio gráfico"
+    ID_APPLICATION_PAGE_SWITCHTO "Coloca a tarefa em primeiro plano e alterna o foco para ela"
```

Comment:
This sentence construction is Brazilian Portuguese.
Maybe it's better to do the translation into Brazilian Portuguese